aiida_mlip/calculations/geomopt.py

```diff
@@ -0,0 +1,114 @@
+"""Class to run geom opt calculations."""
+
+from aiida.common import datastructures
+import aiida.common.folders
+from aiida.engine import CalcJobProcessSpec
+import aiida.engine.processes
+from aiida.orm import Bool, Float, SinglefileData, Str, StructureData, TrajectoryData
+
+from aiida_mlip.calculations.singlepoint import Singlepoint
+
+
+class GeomOpt(Singlepoint):  # numpydoc ignore=PR01
+    """
+    Calcjob implementation to run geometry optimization calculations using mlips.
+
+    Methods
+    -------
+    define(spec: CalcJobProcessSpec) -> None:
+        Define the process specification, its inputs, outputs and exit codes.
+    prepare_for_submission(folder: Folder) -> CalcInfo:
+        Create the input files for the `CalcJob`.
+    """
+
+    _DEFAULT_TRAJ_FILE = "aiida-traj.xyz"
+
+    @classmethod
+    def define(cls, spec: CalcJobProcessSpec) -> None:
+        """
+        Define the process specification, its inputs, outputs and exit codes.
+
+        Parameters
+        ----------
+        spec : `aiida.engine.CalcJobProcessSpec`
+            The calculation job process spec to define.
+        """
+        super().define(spec)
+
+        # Additional inputs for geometry optimization
+        spec.input(
+            "traj",
+            valid_type=Str,
+            required=False,
+            default=lambda: Str(cls._DEFAULT_TRAJ_FILE),
+            help="Trajectory file name",
+        )
+        spec.input(
+            "fully_opt",
+            valid_type=Bool,
+            required=False,
+            default=lambda: Bool(False),
+            help="Whether to optimize the cell as well as atomic positions.",
+        )
+        spec.input(
+            "vectors_only",
+            valid_type=Bool,
+            required=False,
+            default=lambda: Bool(False),
+            help="Whether to allow only hydrostatic deformations.",
```

Comment:
I think you probably took these descriptions from an earlier version of the janus CLI, before suggestions in the review to clarify these help messages. May be worth updating